cmd/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"github.com/nyttikord/gomath"
+	"os"
+	"strings"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Printf("Usage: %s <subcommand>\nUse %s help to get the help.\n", os.Args[0], os.Args[0])
+		os.Exit(1)
+	}
+	subcommand := os.Args[1]
+	switch subcommand {
+	case "help":
+		if len(os.Args) != 2 {
+			fmt.Printf("Help does not take any arguments.\n")
+			os.Exit(1)
+		}
+		fmt.Printf("Subcommands:\n- eval <expression> -> evaluate an expression.\n- latex <expression> -> latexify an expression.\n")
```

Comment:
Add usage and help subcommand to help text.
```suggestion
		fmt.Printf("Usage: %s <subcommand>\nSubcommands:\n- eval <expression> -> evaluate an expression.\n- help -> print this help text\n- latex <expression> -> convert an expression to Latex code.\n")
```